src/vs/editor/contrib/hover/browser/hover.ts

```diff
@@ -222,7 +222,9 @@ export class ModesHoverController implements IEditorContribution {
 
 		this._hoverClicked = false;
 		this._glyphWidget?.hide();
-		this._contentWidget?.hide();
+		if (!this._contentWidget?.widget.isResizing) {
+			this._contentWidget?.hide();
```

Comment:
Can we add a comment explaining why the widget doesn't hide while resizing. Or better explain why we being hidden while resizing?

Reply:
Perhaps it is better to not call _hideWidgets at all when we are resizing, instead I am doing the isResizing check in the various places where _hideWidgets is called, in the onMouseLeave and onMouseMove events